hoprd/hoprd/src/main.rs

```diff
@@ -160,10 +160,10 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
     #[cfg(all(feature = "runtime-tokio", not(feature = "runtime-async-std")))]
     let res = tokio::runtime::Builder::new_multi_thread()
         .enable_all()
-        .thread_stack_size(2 * 1024 * 1024)
+        .max_blocking_threads(1024)
```

Comment:
This will affect our throughput though. Pls test under the load.

Reply:
It should improve the throughput since the default is 512, while under load I would expect we might need more.

Reply:
I removed the custom configuration. I don't think it was required really.